config/install/image.style.media_library_item.yml

```diff
@@ -0,0 +1,6 @@
+langcode: en
+status: true
+dependencies: {  }
+name: media_library_item
+label: 'Media library item'
+effects: {  }
```

Comment:
I just can't imagine the library to work (let alone be snappy) if you display original files (e.g. 10MB in size).